### ISSUE TITLE
`-daemon` crash

### ISSUE THREAD

**Issue by AbcSxyZ** [dogecoin/docker#24]
Option `-daemon` do not work and is crashing with the following error:
```
$ docker run -it dogecoin -daemon
2021-12-04 00:35:31 
2021-12-04 00:35:31 Dogecoin version v1.14.4
2021-12-04 00:35:31 InitParameterInteraction: parameter interaction: -whitelistforcerelay=1 -> setting -whitelistrelay=1
2021-12-04 00:35:31 Assuming ancestors of block 77e3f4a4bcb4a2c15e8015525e3d15b466f6c022f6ca82698f329edef7d9777e have valid signatures.
Dogecoin server starting
```

Any idea if the daemon fork himself and kill its parent to daemonize ? Which may stop the container.

It's what I'm having in mind as a first thing to inspect. I'm not sure about it, but I suspect buggy behavior related to daemonization.

**Comment by AbcSxyZ** [comment]
Tested on [ruimarinho/bitcoin-core/](https://hub.docker.com/r/ruimarinho/bitcoin-core/), it's crashing too.

**Comment by patricklodder** [comment]
`-daemon` needs a root process e.g. `init` - so that won't work?

**Comment by AbcSxyZ** [comment]
Yeah we are having this kind of issue, without a running process as pid 1, it's stopping. Depend on the behavior of `-daemon`.

I'm not sure about the purpose of using a daemonized executable in a container, not really meaningful.

**Comment by xanimo** [comment]
yes my mistake. looks good to me.

**Comment by AbcSxyZ** [comment]
It's interesting to notice, but I'm not sure if and how we should handle this case.

**Comment by patricklodder** [comment]
The only way to handle it is imho to disable `-daemon`. Problem is it can still be thrown into a config file. So maybe just document it?

**Comment by xanimo** [comment]
agree with @patricklodder on second thought. i don't think it's necessary since docker itself can run as daemon using -d. my mistake.

**Comment by AbcSxyZ** [comment]
I was thinking about it and how we should deal with this error. I had an incredible idea : doing nothing.

Regarding the other bitcoin repo, no one is raising issues related to this problem. We may assume users won't really try to use the `-deamon` option with images, advanced user who use more complex settings may understand the implication or find this PR, and at worst few people can raise an issue.

We need to hope people won't get stuck with an old `dogecoin.conf` containing `deamon` setting, it may definitively be confusing where the error is coming from.

Maybe we can leave it like it is, and if people are raising issues related to this point we can adapt the strategy and eventually add it to the doc ? This PR could be a sufficient documentation about this error.

**Comment by patricklodder** [comment]
Yes, I think we should just keep this issue open for now and keep it in the back of our heads that at some point we may need to do something about it. For now, it's easily negated.
